app/models/like.rb

```diff
@@ -0,0 +1,12 @@
+class Like < ApplicationRecord
+  belongs_to :users
```

Comment:
- [ ] Kindly take a look at your association the belongs_to should be to the author also add the class name eg ```belongs_to :foreign_key, class_name: 'User'```

Reply:
I have renamed the author_id column to user_id, therefore, by default, it belongs to the user_id, and does not need any foreign key constraint, Kindly guide, please.

Reply:
@for-ashraf I agree with you. It's not necessary to rename to author_id, user_id will work fine.